k8s/prod/common/sscs/sscs-evidence-share.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: sscs-evidence-share
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: sscs-evidence-share
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-evidence-share
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
```

Comment:
missing the values from Luigi's PR:
https://github.com/hmcts/cnp-flux-config/blob/ced57794baf0bf6ee726ac2c07b2584cfc8a1c36/k8s/aat/common/sscs/sscs-evidence-share.yaml#L21-L24